### PR TITLE
[16.04] Bump Pulsar client lib version number and add timeout options to Pulsar runner

### PR DESCRIPTION
**Pull request by natefoo**
Fixes a timeout bug that kills the handlers on usegalaxy.org. Pulsar side of the fix was at galaxyproject/pulsar#115

### CODE REVIEW

File: lib/galaxy/jobs/runners/pulsar.py [hunk 1]
```diff
@@ -63,6 +64,10 @@
         valid=specs.is_in("urllib", "curl", None),
         default=None
     ),
+    transport_timeout=dict(
+        map=lambda val: None if val == "None" else int(val),
```
Comment (jmchilton) [inline lib/galaxy/jobs/runners/pulsar.py:68]: I'd prefer to see `val.lower() == "none"` - we use that other places. I'm fine if you don't want to switch this though.

Comment (natefoo) [inline lib/galaxy/jobs/runners/pulsar.py:68]: I'm fine changing this, but it's used in other pulsar runner params as well and I probably shouldn't change these for a stable PR?

Comment (jmchilton) [inline lib/galaxy/jobs/runners/pulsar.py:68]: Excellent point!

Comment (natefoo) [inline lib/galaxy/jobs/runners/pulsar.py:68]: Let's accept as is and I'll submit a second PR once it's merged to dev and fix it there?

**Comment by natefoo** [comment]
@galaxybot test this

**Pull request merged**